sdk/core/Azure.Core/samples/Serialization.md

```diff
@@ -1,34 +1,47 @@
-# Azure.Core public serialization samples
+# Azure.Core Public Serialization Guide
 
-## Using explicit cast
+The latest updates to the `Azure.Core` library have simplified serializing and deserializing of all public Azure models. With the addiion of the `ModelSerializer` class, we provide a public interface that exposes the currently internal serialization code so customers can access it without needing to use reflection or write their own translation. The recommended serialization approach is to use the static `ModelSerializer` class. The `ModelSerializer` class also has the `ModelSerializerOptions` which allows the user to set the serialization `ModelSerializerFormat` type (`Json` or `Wire`) and provide custom Serializer types for specific models.
+
+## Key Concepts
+
+- [Using the ModelSerializer](#using-the-modelserializer)
+- [Using explicit cast](#using-explicit-cast)
+- [Using the ModelJsonConverter](#using-the-modeljsonconverter)
+- [Envelope BYOM Case](#envelope-byom-case)
```

Comment:
annelo-msft 

This feels like shorthand that people on our team understand that might be hard for users of the interfaces to decode initally. For public-facing documentation, maybe a different title could help readers?